library/asyncio-llapi-index.po

```diff
@@ -863,124 +898,134 @@ msgstr "``callback`` :meth:`eof_received() <Protocol.eof_received>`"
-msgstr "``callback`` :meth:`buffer_updated() <BufferedProtocol.buffer_updated>`"
+msgstr ""
+"``callback`` :meth:`buffer_updated() <BufferedProtocol.buffer_updated>`"
 
 #: ../../library/asyncio-llapi-index.rst:449
 msgid "Called when the buffer was updated with the received data."
-msgstr ""
+msgstr "在以接收到的資料更新緩衝區時被呼叫。"
 
 #: ../../library/asyncio-llapi-index.rst:451
 msgid "``callback`` :meth:`eof_received() <BufferedProtocol.eof_received>`"
 msgstr "``callback`` :meth:`eof_received() <BufferedProtocol.eof_received>`"
 
 #: ../../library/asyncio-llapi-index.rst:455
 msgid "Datagram Protocols"
-msgstr ""
+msgstr "資料單元協定"
 
 #: ../../library/asyncio-llapi-index.rst:460
 msgid ""
 "``callback`` :meth:`datagram_received() <DatagramProtocol.datagram_received>`"
 msgstr ""
+"``callback`` :meth:`datagram_received() <DatagramProtocol.datagram_received>`"
 
 #: ../../library/asyncio-llapi-index.rst:462
 msgid "Called when a datagram is received."
-msgstr ""
+msgstr "在接收到資料單元時被呼叫。"
 
 #: ../../library/asyncio-llapi-index.rst:464
 msgid "``callback`` :meth:`error_received() <DatagramProtocol.error_received>`"
-msgstr "``callback`` :meth:`error_received() <DatagramProtocol.error_received>`"
+msgstr ""
+"``callback`` :meth:`error_received() <DatagramProtocol.error_received>`"
 
 #: ../../library/asyncio-llapi-index.rst:465
 msgid ""
 "Called when a previous send or receive operation raises an :class:`OSError`."
-msgstr ""
+msgstr "在前一個傳送或接收操作引發 :class:`OSError` 時被呼叫。"
 
 #: ../../library/asyncio-llapi-index.rst:469
 msgid "Subprocess Protocols"
-msgstr ""
+msgstr "子行程協定"
 
 #: ../../library/asyncio-llapi-index.rst:474
 msgid ""
 "``callback`` :meth:`pipe_data_received() <SubprocessProtocol."
 "pipe_data_received>`"
 msgstr ""
+"``callback`` :meth:`pipe_data_received() <SubprocessProtocol."
+"pipe_data_received>`"
 
 #: ../../library/asyncio-llapi-index.rst:476
 msgid ""
 "Called when the child process writes data into its *stdout* or *stderr* pipe."
-msgstr ""
+msgstr "在子行程向 *stdout* 或 *stderr* pipe 寫入資料時被呼叫。"
 
 #: ../../library/asyncio-llapi-index.rst:479
 msgid ""
 "``callback`` :meth:`pipe_connection_lost() <SubprocessProtocol."
 "pipe_connection_lost>`"
 msgstr ""
+"``callback`` :meth:`pipe_connection_lost() <SubprocessProtocol."
+"pipe_connection_lost>`"
 
 #: ../../library/asyncio-llapi-index.rst:481
 msgid ""
 "Called when one of the pipes communicating with the child process is closed."
-msgstr ""
+msgstr "在與子行程通訊的其中一個 pipes 關閉時被呼叫。"
 
 #: ../../library/asyncio-llapi-index.rst:484
 msgid ""
 "``callback`` :meth:`process_exited() <SubprocessProtocol.process_exited>`"
 msgstr ""
+"``callback`` :meth:`process_exited() <SubprocessProtocol.process_exited>`"
 
 #: ../../library/asyncio-llapi-index.rst:486
 msgid "Called when the child process has exited."
-msgstr ""
+msgstr "在子行程退出後被呼叫。"
 
 #: ../../library/asyncio-llapi-index.rst:490
 msgid "Event Loop Policies"
-msgstr ""
+msgstr "事件迴圈 Policies"
```

Comment:
Since it's a special term in Python asyncio like `Future` and `Task`, the term `policy` is maintained (not translated).